_projects/git_task_list.md

```diff
@@ -17,9 +17,7 @@ requirements:
   - "The applicant should have at least one patch merged to any of the coala repositories."
   - "Familiarity with Git, GitHub API, Phabricator API, and GitLab API"
   - "Familiarity with EmberJS and GitHub Pages"
-status:
-  - in_progress
-  - disabled
+status: disabled
```

Comment:
please undo this change.

Someone else has already marked it as disabled, and it is also 'in progress' - this is a strange one ;-)